packages/calling/src/common/Utils.ts

```diff
@@ -1440,10 +1443,11 @@ function isValidServiceDomain(serviceData: ServiceData): boolean {
  * @param serviceData - Input service data to be validated.
  */
 export function validateServiceData(serviceData: ServiceData) {
+  const allowedValues = Object.values(ServiceIndicator);
+  const formattedValues = allowedValues.join(', ').replace(/,([^,]*)$/, ' and$1');
+
   if (!isValidServiceIndicator(serviceData.indicator)) {
-    throw new Error(
-      `Invalid service indicator, Allowed values are: ${Object.values(ServiceIndicator)}`
```

Comment:
Since the types have already been updated with guest calling, why did we need this change to join them? Earlier also we had 2 values.

Reply:
To address this comment I added this change - https://github.com/webex/webex-js-sdk/pull/3784#discussion_r1723545035/